package.json

```diff
@@ -18,7 +18,9 @@
   ],
   "license": "BSD-3-Clause",
   "dependencies": {
+    "adm-zip": "0.4.7",
     "bcryptjs": "2.3.0",
+    "bluebird": "3.4.6",
```

Comment:
We've removed it, but I think we should think about including it in the project. Take a look in the code the additional lines we had to include in order to manage the promises concurrency (probably not as effective as Bluebird does).